### ISSUE TITLE
Add `invalid_images` list to schema, and change `failed_images`

### ISSUE THREAD

**Issue by drsharp** [ifp/schemas#58]
So the `failed_images` schema: https://github.com/ifp/schemas/blob/master/json/internal/property/failed_images-schema_v1.0.0.json

It should now have these fields:

```
      "listing_position",
      "title",
      "cloudinary_account",
      "original_url",
      "failure_message"
```

With `"failure_message"` being a String.

And you should add an `invalid_images` schema right before it on the advert schema: https://github.com/ifp/schemas/blob/master/json/internal/internal_sale-advert-schema_v1.1.0.json

It should be almost the same as `failed_images` is today:

```
      "listing_position",
      "title",
      "original_url",
      "response_code",
      "response_message",
      "response_duration"
```

Basically what is in the current `failed_images` schema except no `cloudinary_account` because it's not related to uploading.

I made a patch to do all this but I don't have access rights to create a Pull Request.

I also don't know: by changing `failed_images` and adding `invalid_images` does that bump the whole internal schema to 1.2.0?

**Comment by drsharp** [comment]
@ifpingram FYI...

**Comment by ifpingram** [comment]
Hi Dan - sorry you had read only. I've now given you write access for future patches, but I'll get @andzandz to make the changes, as we need to roll it out across our servers downstream. Cheers

**Comment by andzandz** [comment]
This is done and deployed to the production Loader.

**Comment by drsharp** [comment]
Thanks for doing this. I'll get the importer code squared away shortly.
